src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerGenotypingEngine.java

```diff
@@ -520,7 +520,7 @@ private VariantContext addNonRefSymbolicAllele(final VariantContext mergedVC) {
     }
 
 
-    private Map<String, List<GATKRead>> overlappingFilteredReads(final Map<String, List<GATKRead>> perSampleFilteredReadList, final SimpleInterval loc) {
+    private static Map<String, List<GATKRead>> overlappingFilteredReads(final Map<String, List<GATKRead>> perSampleFilteredReadList, final SimpleInterval loc) {
```

Comment:
`static` here is essentially an annotation that this method doesn't depend on the particular instance of `HaplotypeCallerGenotypingEngine`, which is a useful hint to anyone reading the code.  More generally, I think that any method that *can* be declared `static` *should* be declared `static`, kind of like `final`.